CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+# Contributing to Thundernetes
```

Comment:
do we need to add a link from main README to this file?

Reply:
it certainly is a good idea!